src/freenet/l10n/freenet.l10n.en.properties

```diff
@@ -1347,6 +1347,8 @@ NodeIPDectector.ipOverride=IP address override
 NodeIPDectector.ipOverrideLong=IP address override (not usually needed) - set this if you have a *static* IP address or a domain name (e.g. via dyndns), and you are behind a firewall. Note that Freenet will still include other detected IP addresses unless you use the "bind to" option.
 NodeIPDectector.tempAddressHint=Temporary IP address hint
 NodeIPDectector.tempAddressHintLong=Temporary hint to what our IP might be; deleted after use
+NodeIPDetector.allowBindToLocalhost=Assume localhost to be routable
+NodeIPDetector.allowBindToLocalhostLong=In Opennet mode it would be impossible for someone else to connect to your node using the localhost loopback address. This feature is needed if you use other applications that provides a reachable address of any sort and proxies that to localhost. Currently this node has no way to guess those kind of addresses and add them to your physical.udp line, and there might be no other peers that can reach certain type of addresses. Activate this only if you know what you are doing.
```

Comment:
I would start this with the second sentence and maybe work the first one into it… at first glance, the first sentence has nothing to do with the configuration property; only after the second sentence it become clear what this means.

Reply:
I left out that sentence completely — the third says it clear enough.